app/controllers/admins_controller.rb

```diff
@@ -7,14 +7,6 @@ def show
       @orders = Order.all
     end
   end
-  
-  # def show
-  #   if current_user == nil || current_user.role != "admin"
-  #     redirect_to "/error404"
-  #   else
-  #     @user = User.find(session[:user_id])
-  #   end
-  # end
```

Comment:
![](https://media0.giphy.com/media/ac7MA7r5IMYda/giphy.gif?cid=5a38a5a23d58084b70e4ffe47681fbfe0e54c94f053e1eca&rid=giphy.gif)

Reply:
destroy it